packages/coinstac-common/src/services/classes/db-registry.js

```diff
@@ -188,14 +188,15 @@ class DBRegistry {
     conf.pouchConfig.ajax = result(this, 'ajax');
 
     /**
-     * Ensure local (client) instances' non-projects databases are stored in
-     * memory.
+     * Ensure local (client) instances' databases are stored in
+     * memory. This excludes projects and local consortium results.
      *
      * {@link https://github.com/MRN-Code/coinstac/issues/155}
      */
     if (
       this.localStores &&
       this.localStores.indexOf('projects') > -1 &&
+      this.localStores.indexOf('local-consortium') > -1 &&
```

Comment:
Yep, `RemotePipelineRunnerPool` grabs the `local-consortium-*` database for runner instantiation: https://github.com/MRN-Code/coinstac/blob/941a7ea50801aaf7e3df0e604d3f27b4c6c14874/packages/coinstac-common/src/models/pipeline/runner/pool/remote-pipeline-runner-pool.js#L31-L63. This makes sense: the server needs access to clients' results (documents in `local-consortium-*` database) to provide them to computations. Also, clients are configured to sync `local-consortium-*` out: https://github.com/MRN-Code/coinstac/blob/941a7ea50801aaf7e3df0e604d3f27b4c6c14874/packages/coinstac-common/src/services/db-registry.js#L58.